site/layouts/partials/home/masthead.html

```diff
@@ -1,23 +1,22 @@
 <div class="bd-masthead mb-3" id="content">
   <div class="container-xxl bd-gutter">
     <div class="col-md-8 mx-auto text-center">
```

Comment:
I'd make this part take at least one screen height for design purpose. Maybe it can be done afterwards (otherwise I trust you on this).

Reply:
Fixed by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2591/commits/b4a3a0f44cfcf3633d4a59917bb5b22fa3c16e29